README.md

```diff
@@ -1,107 +1,41 @@
-Chitter Challenge
-=================
+# Chitter!
```

Comment:
Really nice README. You have added clear instructions and a nice screenshot to show exactly how your app works. You could include details of cloning your file such as (git clone https://github.com/jebax/chitter-challenge.git).